Sprout/Models/Task.h

```diff
@@ -0,0 +1,23 @@
+//
+//  Task.h
+//  Sprout
+//
+//  Created by Sarah Wang on 7/13/21.
+//
+
+#import <Parse/Parse.h>
+#import <Foundation/Foundation.h>
+#import "Goal.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Task : PFObject<PFSubclassing>
+
+@property (nonatomic, strong) NSDate *createdAt;
+@property (nonatomic, strong) NSDate *expiresAt;
+@property (nonatomic) BOOL *completed;
```

Comment:
I actually think an enum might be better here. TaskState { Pending, InProgress, Complete } or something like that. It'll give you more flexibility.